pages/c/_cluster/explorer/index.vue

```diff
@@ -359,6 +367,18 @@ export default {
         <span><LiveDate :value="currentCluster.metadata.creationTimestamp" :add-suffix="true" :show-tooltip="true" /></span>
       </div>
       <div :style="{'flex':1}" />
+      <!--  -->
+      <div>
+        <a
+          class="badge-install"
+          @click="customBadgeDialog"
+        >
+          <i class="icon icon-star" />
```

Comment:
Ask Lauren for an icon for a cluster badge

Reply:
I believe icon-cluster would work. Unless you want something more specific that illustrates a badge.